src/ol/geom/multipoint.js

```diff
@@ -173,7 +173,7 @@ ol.geom.MultiPoint.prototype.intersectsExtent = function(extent) {
  */
 ol.geom.MultiPoint.prototype.setCoordinates = function(coordinates, opt_layout) {
   if (!coordinates) {
-    this.setFlatCoordinates(ol.geom.GeometryLayout.XY, null);
+    this.setFlatCoordinates(ol.geom.GeometryLayout.XY, []);
```

Comment:
Yes. The EMPTY tests does not pass otherwise.
Could be handled in `getCoordinates` instead. That code does not handle that this.flatCoordinates is null and throws.